packages/connect/src/api/signTransaction.ts

```diff
@@ -137,6 +135,11 @@ export default class SignTransaction extends AbstractMethod<'signTransaction', P
         this.params.options = enhanceSignTx(this.params.options, coinInfo);
     }
 
+    get info() {
+        const coinInfo = getBitcoinNetwork(this.payload.coin);
```

Comment:
Here we're sure that `coinInfo` is present? Meaning, the original info message `'Sign transaction'` was unused?

Reply:
yes, this getLabel utility does the job.